lms/djangoapps/teams/tests/test_views.py

```diff
@@ -163,6 +163,14 @@ def setUp(self):
         self.test_team_4 = CourseTeamFactory.create(name='Coal Team', course_id=self.test_course_1.id, is_active=False)
         self.test_team_5 = CourseTeamFactory.create(name='Another Team', course_id=self.test_course_2.id)
 
+        self.test_team_name_id_map = {team.name: team for team in (
```

Comment:
I wish I didn't have to do this, but since team_id is generated dynamically, I couldn't think of any other way to verify it on line 738. Is there some other way I could include the test_team_N.team_id value in ddt.data for those tests?